conf/test.config

```
@@ -19,6 +19,7 @@ params {
   rawcounts = 'https://raw.githubusercontent.com/qbic-pipelines/rnadeseq/dev/testdata/merged_gene_counts.txt'
   model = 'https://raw.githubusercontent.com/qbic-pipelines/rnadeseq/dev/testdata/design.txt'
   genelist = 'https://raw.githubusercontent.com/qbic-pipelines/rnadeseq/dev/testdata/requested_genes.txt'
+  logFCthreshold = 2
```

Here, we need to discuss with Stefan, but I think by default we wanted not to apply any logFC threshold, therefore it should be 0.

```suggestion
  logFCthreshold = 0
```

---

Hi, sure I'll change it back. It's just that applying the threshold enables much faster testing. Would it be an option to create another `test_logFCthreshold.config` file for that or apply the threshold in another test config file?